SU-HSDO/suhumsci
# feat: Add styles for UI People Pattern

#  [JIRA STN-146](https://sparkbox.atlassian.net/browse/STN-146) READY FOR REVIEW

## Summary
This PR implements styles to support a custom People view. It also includes the addition of a conditional statement to the structured card template which prevents HTML from the two column section from rendering if no content for that section is entered in the Drupal admin panel.

The work in this PR has had an initial WIP review with a designer. A second/final review will be completed Tuesday morning.

## Need Review By (Date)
ASAP 2/11/2020

## Urgency
🔥 High

## Steps to Test
1. Run `npm run test` and verify that all tests pass.
2. Create a new person or modify an existing person to include the following:
    - Name
    - Image
    - Bio
    - Title & Department
    - Degrees & Education
    - Fields of Interest
    - Contact Information (email, phone, etc)
3. Duplicate the existing people view, use the structured card pattern
4. Map fields:
    - Name -> Title
    - Image -> Image
    - Bio -> Description
    - Title & Department -> Subtitle
    - Degrees & Education -> Category
    - Fields of Interest -> Category
    - Contact Information (email, phone, etc) -> Columns (these fields require labels)
5. Create a basic page which uses the people view
6. Confirm the styles implemented reflect the designs in [Figma](https://www.figma.com/file/toNZnFLadtpNpG5dPzNGWz/⚙%EF%B8%8F-Colorful-Theme---Working?node-id=2288%3A1396)
7. Since there were changes made to the styles in `hb-card` which is use by other cards, review a horizontal card list page and an events list page to confirm that the cards continue to display as expected.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
